rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -810,12 +816,25 @@ def _get_transformer_dimensions(
         return transformer_layers, transformer_units
 
     def _prepare_transformer(
-        self, attribute: Text, config: Dict[Text, Any]
+        self,
+        attribute: Text,
+        attribute_signature: Dict[Text, List[FeatureSignature]],
+        config: Dict[Text, Any],
     ) -> Tuple[int, int]:
         """Creates a transformer & returns its number of layers and output units."""
         transformer_layers, transformer_units = self._get_transformer_dimensions(
             attribute, config
         )
+        if transformer_layers > 0:
+            if SEQUENCE not in attribute_signature:
+                raise TFLayerConfigException(
+                    "The attribute signature must contain some sequence feature"
+                    "signatures since the transformer layers expects sequential input"
+                    "but none were found."
```

Comment:
```suggestion
                    f"The attribute signature must contain signature for {SEQUENCE} attribute"
                    "since the transformer layers expects sequential input"
                    "but no signature was found."
```

Reply:
"no signature" found sounds like the whole attribute signature is missing, isn't it?

Reply:
Updated the suggestion.